### PR TITLE
V3/ Add wui-card-select component

### PR DESCRIPTION
**Pull request by svenvoskamp**
# Changes

- feat: Add `wui-card-select` component

### CODE REVIEW

File: packages/ui/src/composites/wui-card-select/index.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  public render() {
+    return html`
+      <button ?disabled=${this.disabled}>
+        <wui-wallet-image size="md" src=${this.imageSrc} alt=${this.name}></wui-wallet-image>
+        <wui-text variant="tiny-500" color="inherit">${this.getTruncatedName()}</wui-text>
+      </button>
+    `
+  }
+
+  private getTruncatedName() {
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-card-select/index.ts:30]: This is a good candidate to make into utility inside ui package i.e. `HelperUtil` or `ToolsUtil`, you can reference how v2 does some of these files. Alternatively, can we acheive this with css? i.e https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

File: packages/ui/src/composites/wui-card-select/styles.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { css } from 'lit'
+
+export default css`
+  button {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-card-select/styles.ts:7]: Would this component be responsible or does it actually have fixed width? We need to check with design team, either this button expands or stays constant and only spacing / gaps etc around it do

File: packages/ui/src/composites/wui-card-select/styles.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { css } from 'lit'
+
+export default css`
+  button {
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-card-select/styles.ts:4]: I've noticed that we have quiet a few repeated styles on components that use buttons atm, would it make sense to put them under global styles on a `button`? If not all buttons need this, we could make a global class like `wui-button-base` or something and apply it where needed. 

Similarly for button focus / hover states etc.

File: apps/gallery/stories/composites/wui-card-select.stories.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import '../../components/gallery-container'
+import type { WuiCardSelect } from '@web3modal/ui/src/composites/wui-card-select'
+import { html } from 'lit'
+
+type Component = Meta<WuiCardSelect>
+
+export default {
+  title: 'Composites/wui-card-select',
+  args: {
+    imageSrc:
```
Comment (0xAsimetriq) [inline apps/gallery/stories/composites/wui-card-select.stories.ts:12]: Could reuse that `walletImagePresets[0].src` trick from last pr here as well, maybe there are other places?

**Review by 0xAsimetriq — CHANGES_REQUESTED** [review]
Make sure to update with latest V3 as well, there are new dangerjs rules that should catch one other issue here where new components need to be added to `packages/ui/index.ts` file. Would be good to test it before fixing. (just re-run pr check when updated)

File: packages/ui/src/composites/wui-card-select/index.ts (hunk 1 above)
Comment (svenvoskamp) [reply packages/ui/src/composites/wui-card-select/index.ts:30]: Went for the `css` solution. Cleaner imo.

**Review by 0xAsimetriq — APPROVED** [review]

**Pull request merged**